postgres/scripts/v2_to_v3/05_update_results_user_groups_table.sql

```diff
@@ -0,0 +1,31 @@
+/*
+ * This script updates and tra the data from the 'results' table
+ * and transfers them to the new tables
+ * - 'mapping_sessions' and
+ * - 'mapping_sessions_results'
+ */
+SET search_path = 'public';
+
+-- Create new user_groups mapping sessions table
+CREATE TABLE IF NOT EXISTS mapping_sessions_user_groups (
+    mapping_session_id int8,
+    user_group_id varchar,
+    PRIMARY KEY (mapping_session_id, user_group_id),
+    FOREIGN KEY (mapping_session_id) REFERENCES mapping_sessions (mapping_session_id),
+    FOREIGN KEY (user_group_id) REFERENCES user_groups (user_group_id)
+);
+
+
+-- Copy data to new table
+INSERT INTO mapping_sessions_user_groups
+(
+    SELECT
+        MS.mapping_session_id,
+        rug.user_group_id
+    FROM results_user_groups rug
```

Comment:
can you check if this table still exists? I think we now use the `mapping_sessions_user_groups` table instead.

Reply:
ah but maybe I'm confused. where do you define the `results_user_groups` table. Maybe I've missed it.

Reply:
ah I think now I have it, this is the existing results_user_groups table which we already have in the dev server, but not on production.

Reply:
Yes, for migration for the dev server.